### DISCUSSION TITLE
Creating a team project through API

### DISCUSSION THREAD

**Discussion by kr2792** [vercel/vercel#6350] [Help] [answered]
I've been experimenting with creating a project through the create-project [API ](https://vercel.com/docs/api#endpoints/projects/create-a-project) and it works well for my personal account. Is there a way for me to create a project and give it directly to a team? I'm using a token created on my own user, but I can't see that there's any option for getting a token from a team. 

This is the API function I'm using. Is there there perhaps a way that I can assign a team of mine here?

```
export default async function (req, res) {
  const { projectName } = req.body

  try {
    const projectResponse = await fetch("https://api.vercel.com/v8/projects/", {
      method: 'POST',
      headers: {
        'Content-Type': 'application/json',
        'Authorization': `Bearer ${process.env.NEXT_PUBLIC_VERCEL_BEARER_TOKEN}`
      },
      body: JSON.stringify({
        "name": projectName,
        "gitRepository": {
          "type": "github",
          "repo": "tokni/fantilsi"
        }

      })
    })

    res.status(projectResponse.status).send(projectResponse.statusText)

  } catch (error) {
    res.status(400).send("error")
  }

}

```

**Comment by Praneeth-Pike** [comment, marked as answer]
Hey you need to append `?teamId=""` to your endpoint if you want to create for a team. 

The documentation is here: [https://vercel.com/docs/api#api-basics/authentication/accessing-resources-owned-by-a-team](https://vercel.com/docs/api#api-basics/authentication/accessing-resources-owned-by-a-team)

**Reply by kr2792** [reply]
Thank you! This was it. :)
